src/tests/apps.cpp

```diff
@@ -55,6 +55,66 @@ void annotatedTextSentences(Ptr<Options> options, bool source) {
   }
 }
 
+void translationCache(Ptr<Options> options) {
+  // Prepare memories for bytearrays (including model, shortlist and vocabs)
+  MemoryBundle memoryBundle;
+  ResponseOptions responseOptions;
+
+  if (options->get<bool>("bytearray")) {
+    // Load legit values into bytearrays.
+    memoryBundle = getMemoryBundleFromConfig(options);
+  }
+
+  Service service(options, std::move(memoryBundle));
+
+  // Read a large input text blob from stdin
+  std::ostringstream inputStream;
+  inputStream << std::cin.rdbuf();
+  std::string input = inputStream.str();
```

Comment:
What is this and why would I want it?

Reply:
So reading the entire file to stdin.  I didn't even know one could do that with a buffer.

Reply:
Moving to an issue outside this PR: https://github.com/browsermt/bergamot-translator/issues/221, where this will be handled, along with other such occurences. Marking as resolved.